docs/05_operations/11_release-notes/02_migration-6-7.md

```diff
@@ -0,0 +1,80 @@
+---
+title: 'Migration guide: version 6.0 to 7.0'
+sidebar_label: 'Migration guide'
+sidebar_position: 2
+id: migration-6-7
+keywords: [operations, release notes, OCT 2023]
+tags:
+- operations
+- release notes
+- migration
+---
+
+Version 7.0.0 of the Genesis platform requires Java version 17 and gradle version 8.3.
+:::info
+If you have an existing application running on version 6 of the Genesis platform and you are using gradle, these instructions will enable you to migrate to version 7 of the platform without difficulties.
+For applications running on older versions of the platform, please contact your Genesis representative.
+:::
+## Upgrade references to the artifacts.
+To migrate an existing application to version 7.0.0:
+1. In your **gradle.properties** file, bump up `genesisVersion` and all modules in the application to 7.0.0.
+```
+genesisVersion=7.0.0
+authVersion=7.0.0
+deployPluginVersion=7.0.0
+```
+## Modify the build
+1. Update your **server/jvm/build.gradle.kts** file so that it references Java 17 and Kotlin 1.9.
```

Comment:
'java 17 (including compiler options)`

See on line 25 in the picture, the text is:
from `-Xjvm-default=enable` to `-Xjvm-default=all`